packages/nx/src/lock-file/yarn.ts

```diff
@@ -33,7 +33,6 @@ export function parseYarnLockFile(lockFile: string): LockFileData {
     isBerry
   );
   const hash = hashString(lockFile);
-  const [mappedPackages, workspacePackages] = mapPackages(dependencies);
```

Comment:
This is a rebase misshap - there were never two instances of this call, otherwise the build would have been failing